packages/web-components/fast-foundation/src/tooltip/stories/tooltip.stories.ts

```diff
@@ -21,28 +17,45 @@ const storyTemplate = html<StoryArgs<FASTTooltip>>`
 export default {
     title: "Tooltip",
     args: {
+        anchor: "anchor-default",
         storyContent: "Tooltip",
     },
     argTypes: {
-        autoUpdateMode: { control: "select", options: Object.values(AutoUpdateMode) },
-        delay: { control: "number" },
-        horizontalViewportLock: { control: "boolean" },
-        position: { control: "select", options: Object.values(TooltipPosition) },
-        verticalViewportLock: { control: "boolean" },
-        visible: { control: "boolean" },
-    },
-    parameters: {
-        docs: { inlineStories: false },
+        anchor: { control: "text" },
+        placement: {
+            control: "select",
+            options: Object.values(TooltipPlacement),
+        },
+        show: { control: "select", options: [undefined, true, false] },
     },
 } as Meta<FASTTooltip>;
 
-export const Tooltip: Story = renderComponent(
+export const Tooltip: Story<FASTTooltip> = renderComponent(html<StoryArgs<FASTTooltip>>`
+    <div>
+        <fast-button id="${x => x.anchor}">
+            ${x => x.buttonContent ?? "Hover or focus me"}
+        </fast-button>
```

Comment:
This is a good example as I think it illustrates an issue we currently have with cross-root ARIA for Button and similar components which delegate focus. While the tooltip shows up, it's not announced, which means that (to my knowledge) aria-describedby isn't working since the two id's are in separate DOM's.

Reply:
TLDR, we need to review if we either can figure out a way to manage this issue or if we should shift how we approach components which currently delegate focus.

Reply:
I had a look at this issue yesterday and experienced inconsistent results with `inert` in Chrome: https://github.com/Vonage/vivid-3/discussions/859